coalib/bears/Bear.py

```diff
@@ -136,6 +146,19 @@ def can_detect(cls):
         """
         return cls.CAN_DETECT | cls.CAN_FIX
 
+    @classproperty
+    def maintainers(cls):
+        """
```

Comment:
ah nvm i noticed i forgot to put those just in rush to see if its working, will fix this in 20-30mins when i got time again